lib/autokey/model.py

```diff
@@ -343,11 +343,17 @@ def get_filter_regex(self):
             return ""
 
     def filter_matches(self, otherFilter):
+        # XXX Should this be and?
```

Comment:
I think this is used to check if the trigger of another item, given that it has this `otherFilter` filter, will collide with the item’s own trigger.
So it does collide if the other filter is None(matching everything) _or_ the own filter is None (matching everything). So `or` should be correct.

Reply:
Ah, I see. It assumes the item has a filter - which makes sense given the class it's in. But possibly this should be made explicit?